types/serve-static/index.d.ts

```diff
@@ -96,9 +91,12 @@ declare namespace serveStatic {
          * path the file path that is being sent
          * stat the stat object of the file that is being sent
          */
-        setHeaders?: (res: express.Response, path: string, stat: any) => any;
+        setHeaders?: (res: R, path: string, stat: any) => any;
+    }
+
+    interface RequestHandler<R extends http.OutgoingMessage> {
+        (request: http.IncomingMessage, response: R, next: () => void): any;
     }
-    function serveStatic(root: string, options?: ServeStaticOptions): express.Handler;
```

Comment:
I'm not sure if this is required or what, seems it's not so I removed it as it looks like a duplicate declaration.

Reply:
I think it may have been added to (incorrectly) model `--esModuleInterop`. `serveStatic` [doesn't assign itself as a property of itself](https://unpkg.com/browse/serve-static@1.14.1/index.js#L28), so removing this is correct.

Reply:
Yeah exactly I realized that later and was about to point that out as I myself had hit a runtime error because `import { serveStatic } from "serve-static"` compiled anyway.